llarp/net/ip_address.hpp

```diff
@@ -24,7 +24,7 @@ namespace llarp
     /// Empty constructor.
     IpAddress() = default;
     /// move construtor
-    IpAddress(IpAddress &&) = default;
```

Comment:
bin this entire file if you can

Reply:
https://github.com/oxen-io/lokinet/releases/tag/v0.9.11